### PR TITLE
2.10.5 & 2.11.6

### PR DESCRIPTION
**Pull request by adriaanm**
review by @sjrd|@heathermiller|@retronym|@gkossakowski

### CODE REVIEW

File: download/index.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 title: Download
-release_version: 2.11.5
+release_version: 2.11.6
 release_date: "January 14, 2015"
 other_releases: [
-  ["maintenance_version", "Current 2.9.x maintenance release", 2.9.3, "February 28, 2013"],
-  ["maintenance_version", "Current 2.10.x maintenance release", 2.10.4, "March 24, 2014"],
-  ["development_version", "Current 2.11.x release", 2.11.5, "January 14, 2015"]
+  ["maintenance_version", "Last 2.9.x maintenance release", 2.9.3, "February 28, 2013"],
+  ["maintenance_version", "Last 2.10.x maintenance release", 2.10.5, "March 5, 2015"],
```
Comment (retronym) [inline download/index.md]: `Final` would be less ambiguous.

**Comment by retronym** [comment]
LGTM

File: download/index.md (hunk 1 above)
Comment (adriaanm) [inline download/index.md]: But we also use final when going from RC to, erm, final :-)

Comment (retronym) [inline download/index.md]: Very true. Let's leave it. ("Ultimate" release doesn't have the right connotation!)

Comment (adriaanm) [inline download/index.md]: Something to consider, though, but we'll need to develop some sbt plugins to maintain the Scala Basic/Home/Essentials/Professional/Ultimate/Massive versions.

**Comment by gkossakowski** [comment]
LGTM

File: download/index.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
 ---
 layout: maindownloadpage
 title: Download
-release_version: 2.11.5
+release_version: 2.11.6
 release_date: "January 14, 2015"
```
Comment (sjrd) [inline download/index.md]: March 5

**Comment by sjrd** [comment]
That's all.

File: download/index.md (hunk 2 above)
Comment (adriaanm) [inline download/index.md]: Good catch! Missed that one.

**Comment by adriaanm** [comment]
fixed date

**Comment by adriaanm** [comment]
Alright, going to hit merge then!

**Pull request merged**